aggie-coding-club/Rev-Registration
# Save schedules

## Description

Adds saving schedules to the session. Note that this is based off of #410 due to its addition of the abstract get/set state route functionality.

## Rationale

I set the throttle time 5 seconds since I don't expect schedules to be modified much, but we can increase it if we really want to.

## How to test

Generate schedules then save (and rename) them, refresh the page, and ensure the schedules were saved.

## Screenshots

![save-schedules-demo](https://user-images.githubusercontent.com/7295783/96643735-a0d82100-12ed-11eb-8274-165061ee3b1e.gif)

## Related tasks

Closes #267
